assets/css/src/components/_datepicker.scss

```diff
@@ -0,0 +1,95 @@
+@import '../../../../node_modules/react-datepicker/src/stylesheets/datepicker';
+
+.mailpoet-datepicker {
+  display: inline-block;
+  vertical-align: middle;
+
+  .react-datepicker {
+    background-color: $color-input-background;
+    border: solid 1px $color-tertiary-light;
+    border-radius: 4px;
+    box-shadow: 0 2px 8px 0 $color-tertiary-light;
+    font-family: $font-family;
+    font-size: $font-size;
+    height: 400px;
+    width: $grid-column;
+  }
+
+  .react-datepicker-wrapper,
+  .react-datepicker-wrapper input {
+    width: 100%;
```

Comment:
`box-sizing: border-box;` should be added, so the width is correctly calculated (and not add the padding resulting in more than 100%)